problems/variables.js

```diff
@@ -53,6 +134,10 @@
 // * a = 123
 // * Output:
 // * 3
+let a = 2603
+
+let finalDigit = a % 2600
```

Comment:
This is correct, but only for your one input case. Think about an answer that could be correct universally